admin-dev/themes/new-theme/scss/components/_form.scss

```diff
@@ -225,3 +225,4 @@
     }
   }
 }
+
```

Comment:
Not really useful 😁

Reply:
AGreed, not really blocking either If there are other things to fix in this PR I'll fix the one as well 😉